changelog/1.3.0.md

```diff
@@ -0,0 +1,37 @@
+# Scala.meta 1.3.0
+
+## Trees
+
+  * Type.Compound has been split into `Type.With` and `Type.Refine` in order to provide common ground for compatibility with Dotty. Dotty gets rid of `with` in favor of intersection types, but it still has type refinements, so we needed to extract `Type.Refine`, so that it can be used in both dialects.
+
+    ```
+    scala> t"A with B".structure
+    res0: String = Type.With(Type.Name("A"), Type.Name("B"))
+
+    scala> t"{ def x: Int }".structure
```

Comment:
I think it's bencause the spec considers them *Refinements*: http://scala-lang.org/files/archive/spec/2.11/03-types.html#compound-types